corgi/tasks/errata_tool.py

```diff
@@ -117,6 +119,9 @@ def update_variant_repos() -> None:
     CDN repos are saved as Channels and are linked to Variants as their children. Multiple Variants
     can link to the same CDN repo.
     """
+    logger.info("Getting active repositories from Pulp")
+    Pulp().get_active_repositories()
```

Comment:
I wonder if calling this method here would make the update_variant_repos() task time out. Maybe it would be better to:
1) Instead of calling update_variant_repos(), call get_active_repositories() first as a Celery task
2) At the end of that Celery task, call update_variant_repos() next